src/dodal/beamlines/vmxm.py

```diff
@@ -0,0 +1,57 @@
+from dodal.beamlines.beamline_utils import device_instantiation
+from dodal.beamlines.beamline_utils import set_beamline as set_utils_beamline
+from dodal.devices.eiger import EigerDetector
+from dodal.devices.fast_grid_scan import FastGridScan
+from dodal.devices.zebra import Zebra
+from dodal.log import set_beamline as set_log_beamline
+from dodal.utils import get_beamline_name, skip_device
+
+SIM_BEAMLINE_NAME = "svmxm"
+
+BL = get_beamline_name(SIM_BEAMLINE_NAME)
+set_log_beamline(BL)
+set_utils_beamline(BL)
+
+
+@skip_device(lambda: BL == SIM_BEAMLINE_NAME)
+def eiger(wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False) -> EigerDetector:
+    """Get the vmxm eiger device, instantiate it if it hasn't already been.
+    If this is called when already instantiated in vmxm, it will return the existing object.
+    """
+    return device_instantiation(
+        device_factory=EigerDetector,
+        name="eiger",
+        prefix="-EA-EIGER-01:",
+        wait=wait_for_connection,
+        fake=fake_with_ophyd_sim,
+    )
+
+
+@skip_device(lambda: BL == SIM_BEAMLINE_NAME)
+def fast_grid_scan(
+    wait_for_connection: bool = True, fake_with_ophyd_sim: bool = False
+) -> FastGridScan:
+    """Get the vmxm fast_grid_scan device, instantiate it if it hasn't already been.
+    If this is called when already instantiated in vmxm, it will return the existing object.
+    """
+    return device_instantiation(
+        device_factory=FastGridScan,
+        name="fast_grid_scan",
+        prefix="-MO-SAMP-11:FGS:",  # TODO: currently needs Pxxxx prefixes in EPICS on VMXm, ask controls if we can alias to versions without these prefixes?
```

Comment:
Have asked controls (Andy) about this, await response before deciding what to do with this

Reply:
We also have an issue where the i03 FGS uses `DWELL_TIME` and VMXm uses `EXPOSURE_TIME`. This could warrant two different ophyd devices, but they should both have a standard `exposure_time` signal, as that's more user friendly. i.e. if you do make two devices rename the i03 one

Reply:
I'll ask Andy whether this interface can be made the same as i03. It would be _nice_ to avoid needing two separate ophyd devices for what is conceptually very similar/the same I think?

Or do you think long term we'll go towards separate ophyd devices _anyway_?

Reply:
Maybe we can create an "alias" in ophyd so that both work?

Reply:
I agree having one device is better. If we have to have two then they should be interchangeable at the plan level. Long term we'll end up getting rid of the FGS on i03 anyway